client/my-sites/email/email-management/home/utils.js

```diff
@@ -130,8 +142,8 @@ export function resolveEmailPlanStatus( domain ) {
 				return defaultWarningStatus;
 			}
 		}
-
-		if ( getMaxTitanMailboxCount( domain ) > getConfiguredTitanMailboxCount( domain ) ) {
+		// Check for unused mailboxes
+		if ( hasUnusedMailboxWarnings( emailAccount?.warnings ) ) {
 			return defaultWarningStatus;
 		}
```

Comment:
This will stop showing this error on the main home page. I think we need to check for the case where we don't have an `emailAccount` despite the data being loaded.
```suggestion
		// Check for unused mailboxes
		if ( emailAccount && hasUnusedMailboxWarnings( emailAccount?.warnings ) ) {
			return defaultWarningStatus;
		}

		// Fallback logic if we don't have an emailAccount - this will initially be the case for the email home page.
		if ( ! isLoadingEmails && ! emailAccount && getMaxTitanMailboxCount( domain ) > getConfiguredTitanMailboxCount( domain ) ) {
			return defaultWarningStatus;
		}
```

Reply:
👍